<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>630</width>
    <height>586</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Engineering Diffraction Analysis - Settings</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QGroupBox {
border: 1px solid grey;border-radius: 10px;margin-top: 1ex; margin-right: 0ex
}
QGroupBox:title {
                          subcontrol-origin: margin;
                          subcontrol-position: top center;
                          padding-top: 0px;
                          padding-bottom: 0px;
							  padding-left: 5px;
                          padding-right: 5px;
                          color: rgb(56, 56, 56)
}</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="group_general">
     <property name="title">
      <string>General</string>
     </property>
     <layout class="QVBoxLayout" name="generalVBox">
      <item>
       <widget class="FileFinderWidget" name="finder_save" native="true"/>
      </item>
      <item>
    <widget class="QGroupBox" name="groupBox_textureVectors">
     <property name="title">
      <string>Texture Directions</string>
     </property>
 <property name="styleSheet">
  <string notr="true">
   QGroupBox {
    border: 1px solid #ccc;
    border-radius: 4px;
    margin-top: 10px;
   }
   QGroupBox::title {
    subcontrol-origin: margin;
    subcontrol-position: top left;
    padding: 0 5px;
    font-weight: normal;
    color: #555;
   }
  </string>
 </property>
     <layout class="QGridLayout">
      <item row="0" column="0">
   <spacer name="spacer_left">
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
    <property name="sizeHint" stdset="0">
     <size>
      <width>40</width>
      <height>20</height>
     </size>
    </property>
   </spacer>
  </item>
         <item row="0" column="1">
       <widget class="QLineEdit" name="lineedit_RD">
               <property name="text">
           <string/>
          </property>
        <property name="maximumWidth">
      <number>30</number>
    </property>
    <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
      </sizepolicy>
    </property>
         </widget>
      </item>
      <item row="0" column="2">
       <widget class="QLineEdit" name="lineedit_RD0">
               <property name="text">
           <string/>
          </property>
        <property name="maximumWidth">
      <number>30</number>
    </property>
    <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
      </sizepolicy>
    </property>
         </widget>
      </item>
      <item row="0" column="3">
       <widget class="QLineEdit" name="lineedit_RD1">
               <property name="text">
           <string/>
          </property>
        <property name="maximumWidth">
      <number>30</number>
    </property>
    <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
      </sizepolicy>
    </property>
         </widget>
      </item>
      <item row="0" column="4">
       <widget class="QLineEdit" name="lineedit_RD2">
               <property name="text">
           <string/>
          </property>
        <property name="maximumWidth">
      <number>30</number>
    </property>
    <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
      </sizepolicy>
    </property>
         </widget>
      </item>
         <item row="1" column="1">
       <widget class="QLineEdit" name="lineedit_ND">
               <property name="text">
           <string/>
          </property>
        <property name="maximumWidth">
      <number>30</number>
    </property>
    <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
      </sizepolicy>
    </property>
         </widget>
      </item>
      <item row="1" column="2">
       <widget class="QLineEdit" name="lineedit_ND0">
               <property name="text">
           <string/>
          </property>
        <property name="maximumWidth">
      <number>30</number>
    </property>
    <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
      </sizepolicy>
    </property>
         </widget>
      </item>
      <item row="1" column="3">
       <widget class="QLineEdit" name="lineedit_ND1">
               <property name="text">
           <string/>
          </property>
        <property name="maximumWidth">
      <number>30</number>
    </property>
    <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
      </sizepolicy>
    </property>
         </widget>
      </item>
      <item row="1" column="4">
       <widget class="QLineEdit" name="lineedit_ND2">
               <property name="text">
           <string/>
          </property>
        <property name="maximumWidth">
      <number>30</number>
    </property>
    <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
      </sizepolicy>
    </property>
         </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="lineedit_TD">
               <property name="text">
           <string/>
          </property>
        <property name="maximumWidth">
      <number>30</number>
    </property>
    <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
      </sizepolicy>
    </property>
         </widget>
      </item>
      <item row="2" column="2">
       <widget class="QLineEdit" name="lineedit_TD0">
               <property name="text">
           <string/>
          </property>
        <property name="maximumWidth">
      <number>30</number>
    </property>
    <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
      </sizepolicy>
    </property>
         </widget>
      </item>
      <item row="2" column="3">
       <widget class="QLineEdit" name="lineedit_TD1">
               <property name="text">
           <string/>
          </property>
        <property name="maximumWidth">
      <number>30</number>
    </property>
    <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
      </sizepolicy>
    </property>
         </widget>
      </item>
      <item row="2" column="4">
       <widget class="QLineEdit" name="lineedit_TD2">
               <property name="text">
           <string/>
          </property>
        <property name="maximumWidth">
      <number>30</number>
    </property>
    <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
      </sizepolicy>
    </property>
         </widget>
      </item>
      <item row="2" column="5">
   <spacer name="spacer_left">
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
    <property name="sizeHint" stdset="0">
     <size>
      <width>40</width>
      <height>20</height>
     </size>
    </property>
   </spacer>
  </item>
     </layout>
    </widget>
   </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="group_calib">
     <property name="title">
      <string>Calibration</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <widget class="FileFinderWidget" name="finder_fullCalib" native="true"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="group_calib">
     <property name="title">
      <string>Absorption Correction</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="1" column="2">
         <widget class="QLineEdit" name="monte_carlo_lineedit">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLabel" name="label_10">
          <property name="text">
           <string>Monte-Carlo Parameters</string>
          </property>
         </widget>
        </item>
      <item row="2" column="2">
         <widget class="QCheckBox" name="abs_corr_checkBox">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QLabel" name="label_10">
          <property name="text">
           <string>Remove Files From ADS after processing</string>
          </property>
         </widget>
        </item>
         <item row="3" column="2">
         <widget class="QCheckBox" name="eulerAngles_checkBox">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QLabel" name="label_10">
          <property name="text">
           <string>Orientation File is Euler Angles</string>
          </property>
         </widget>
        </item>
         <item row="4" column="2">
         <widget class="QLineEdit" name="eulerAngles_lineedit">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="4" column="1">
         <widget class="QLabel" name="label_10">
          <property name="text">
           <string>Euler Angle Scheme</string>
          </property>
         </widget>
        </item>
         <item row="5" column="2">
         <widget class="QLineEdit" name="eulerAnglesSense_lineedit">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="5" column="1">
         <widget class="QLabel" name="label_10">
          <property name="text">
           <string>Euler Angles Sense</string>
          </property>
         </widget>
        </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="fitting">
     <property name="title">
      <string>Sample Logs - Fitting / GSAS II</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
         <item row="0" column="0">
         <widget class="QLabel" name="label_10">
          <property name="text">
           <string>Auto-populate file browser with Texture Files</string>
          </property>
         </widget>
      </item>
      <item row="0" column="1">
       <widget class="QCheckBox" name="textureAutoPopulate"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="log_list_label"/>
      </item>
      <item row="2" column="0">
       <widget class="QListWidget" name="log_list">
        <property name="verticalScrollBarPolicy">
         <enum>Qt::ScrollBarAlwaysOn</enum>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="primary_log_label"/>
      </item>
      <item row="4" column="0">
       <widget class="QComboBox" name="primary_log"/>
      </item>
      <item row="4" column="1">
       <layout class="QVBoxLayout" name="SeqFitOrder_layout">
        <item>
         <widget class="QCheckBox" name="check_ascending">
          <property name="text">
           <string>Ascending</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="check_descending">
          <property name="text">
           <string>Descending</string>
          </property>
          <property name="checked">
           <bool>false</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="gridGroupBox">
     <property name="title">
      <string>Fitting</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_6">
      <property name="leftMargin">
       <number>9</number>
      </property>
      <property name="topMargin">
       <number>9</number>
      </property>
      <property name="rightMargin">
       <number>9</number>
      </property>
      <property name="bottomMargin">
       <number>9</number>
      </property>
      <item row="0" column="0">
       <widget class="QLabel" name="peak_list_label"/>
      </item>
      <item row="1" column="0">
       <widget class="QComboBox" name="peak_list"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="group_calib">
     <property name="title">
      <string>Texture</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
         <widget class="QLabel" name="label_10">
          <property name="text">
           <string>Cost Function Threshold</string>
          </property>
         </widget>
        </item>
      <item row="0" column="1">
         <widget class="QLineEdit" name="cost_func_thresh_lineedit">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="maximumSize">
           <size>
            <width>50</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
      <item row="1" column="0">
         <widget class="QLabel" name="label_10">
          <property name="text">
           <string>Peak Position Threshold</string>
          </property>
         </widget>
        </item>
      <item row="1" column="1">
         <widget class="QLineEdit" name="peak_pos_thresh_lineedit">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="maximumSize">
           <size>
            <width>50</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
          <item row="2" column="0">
         <widget class="QCheckBox" name="expPF_checkBox">
          <property name="text">
           <string>Scatter Plot Experimental Pole Figure</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
         <item row="3" column="0">
         <widget class="QLabel" name="label_10">
          <property name="text">
           <string>Contour Kernel Size</string>
          </property>
         </widget>
        </item>
      <item row="3" column="1">
         <widget class="QLineEdit" name="contourKernel_lineedit">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="maximumSize">
           <size>
            <width>50</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="group_gsas2">
     <property name="title">
      <string>GSAS II</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_4">
      <item row="0" column="0">
       <widget class="FileFinderWidget" name="finder_path_to_gsas2" native="true"/>
      </item>
      <item row="1" column="0">
       <layout class="QGridLayout" name="gridLayout_18" rowstretch="0,0" columnstretch="0,0,0,0,0,0">
        <property name="sizeConstraint">
         <enum>QLayout::SetDefaultConstraint</enum>
        </property>
        <property name="verticalSpacing">
         <number>6</number>
        </property>
        <item row="0" column="1">
         <widget class="QLabel" name="label_10">
          <property name="text">
           <string>Timeout Interval</string>
          </property>
         </widget>
        </item>
        <item row="0" column="3">
         <widget class="QLabel" name="label_11">
          <property name="text">
           <string>seconds</string>
          </property>
         </widget>
        </item>
        <item row="0" column="2">
         <widget class="QLineEdit" name="timeout_lineedit">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="maximumSize">
           <size>
            <width>50</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="0" column="5">
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="1" column="1">
         <widget class="QLabel" name="label_12">
          <property name="text">
           <string>dmin</string>
          </property>
         </widget>
        </item>
        <item row="1" column="2">
         <widget class="QLineEdit" name="dSpacing_min_lineedit">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="maximumSize">
           <size>
            <width>50</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="1" column="5">
         <spacer name="horizontalSpacer_3">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="1" column="3">
         <widget class="QLabel" name="label_16">
          <property name="text">
           <string>Å</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QFrame" name="frame">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="btn_apply">
        <property name="text">
         <string>Apply</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btn_ok">
        <property name="text">
         <string>OK</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btn_cancel">
        <property name="text">
         <string>Cancel</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>FileFinderWidget</class>
   <extends>QWidget</extends>
   <header>mantidqt.widgets.filefinderwidget</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
